src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery[] arrayLottery = new Lottery[3];
```

Comment:
Use one instance of `Lottery`
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
        Lottery lottery = new Lottery();
```